libs/java/client_common/src/main/java/com/yahoo/athenz/common/utils/SSLUtils.java

```diff
@@ -0,0 +1,280 @@
+        }
+        
+        public ClientSSLContextBuilder trustStorePassword(final char[] trustStorePassword) {
+            this.trustStorePassword = trustStorePassword;
+            return this;
+        }
+        
+        public ClientSSLContextBuilder trustStoreType(final String trustStoreType) {
+            this.trustStoreType = trustStoreType;
+            return this;
+        }
+        
+        public ClientSSLContextBuilder keyStorePasswordAppName(final String keyStorePasswordAppName) {
+            this.keyStorePasswordAppName = keyStorePasswordAppName;
+            return this;
+        }
+        
+        public ClientSSLContextBuilder keyManagerPasswordAppName(final String keyManagerPasswordAppName) {
+            this.keyManagerPasswordAppName = keyManagerPasswordAppName;
+            return this;
+        }
+        
+        public ClientSSLContextBuilder trustStorePasswordAppName(final String trustStorePasswordAppName) {
+            this.trustStorePasswordAppName = trustStorePasswordAppName;
+            return this;
+        }
+        
+        public ClientSSLContextBuilder privateKeyStore(final PrivateKeyStore privateKeyStore) {
+            this.privateKeyStore = privateKeyStore;
+            return this;
+        }
+        
+        public ClientSSLContextBuilder certAlias(final String certAlias) {
+            this.certAlias = certAlias;
+            return this;
+        }
+        
+        public SSLContext build() {
+            SSLContext context = null;
+            KeyStore keyStore = null;
+            KeyStore trustStore = null;
+            KeyManagerFactory kmf = null;
+            TrustManagerFactory tmf = null;
+            KeyManager[] keyManagers = null;
+            TrustManager[] trustManagers = null;
+            
+            if (keyStorePath == null && trustStorePath == null) {
+                return null;
+            }
+            
+            try {
+                if (keyStorePath != null) {
+                    LOGGER.info("createSSLContextObject: using SSL KeyStore path: " + keyStorePath);
+                    keyStore = loadStore(keyStorePath, keyStoreType, null, getPassword(keyStorePassword, privateKeyStore, keyStorePasswordAppName));
+                    kmf = KeyManagerFactory.getInstance(KeyManagerFactory.getDefaultAlgorithm());
+                    if (keyManagerPassword == null) {
+                        throw new IllegalArgumentException("Missing key manager password for the key store: " + keyStorePath);
+                    }
+                    keyManagerPassword = getPassword(keyManagerPassword, privateKeyStore, keyManagerPasswordAppName);
+                    kmf.init(keyStore, keyStorePassword);
+                    keyManagers = getAliasedKeyManagers(kmf.getKeyManagers(), certAlias);
+                }
+                if (trustStorePath != null) {
+                    LOGGER.info("createSSLContextObject: using SSL TrustStore path: " + trustStorePath);
```

Comment:
Same here:  We should use {} instead of + so that we don't create the string if the info level is not enabled.